--- jack-audio-connection-kit-0.109.2/configure.ac.orig	2008-02-11 23:41:14.915479704 +0100
+++ jack-audio-connection-kit-0.109.2/configure.ac	2008-02-11 23:50:07.877851499 +0100
@@ -669,11 +669,11 @@
 HAVE_READLINE=true
 AC_CHECK_LIB(readline, readline, [:],
     [unset ac_cv_lib_readline_readline
-     AC_CHECK_LIB(readline, readline, [READLINE_DEPS="-ltermcap"],
+     AC_CHECK_LIB(readline, readline, [READLINE_DEPS="-ltinfo"],
         [unset ac_cv_lib_readline_readline
 	 AC_CHECK_LIB(readline, readline,
 	    [READLINE_DEPS="-lncurses"], [HAVE_READLINE=false], "-lncurses")],
-	"-ltermcap")])
+	"-ltinfo")])
 AC_CHECK_HEADER(readline/chardefs.h, [], [HAVE_READLINE=false])
 if test x$HAVE_READLINE = xfalse; then
 	AC_MSG_WARN([*** the jack_transport example client will not be built])
