--- jack-audio-connection-kit-0.94.0/configure.in.orig	2004-01-13 03:43:12.000000000 +0100
+++ jack-audio-connection-kit-0.94.0/configure.in	2004-01-15 20:05:40.494603552 +0100
@@ -116,7 +116,7 @@
 
 JACK_CORE_CFLAGS="-I\$(top_srcdir) -I\$(top_builddir) -D_REENTRANT -D_POSIX_PTHREAD_SEMANTICS -Wall"
 
-JACK_CFLAGS="$JACK_CORE_CFLAGS -g"
+JACK_CFLAGS="$JACK_CORE_CFLAGS $CFLAGS"
 dnl
 dnl figure out how best to optimize
 dnl
