--- jack1-0.124.1/drivers/freebob/freebob_driver.c.orig	2014-01-22 21:25:41.000000000 +0100
+++ jack1-0.124.1/drivers/freebob/freebob_driver.c	2014-02-01 16:13:41.975609896 +0100
@@ -38,8 +38,8 @@
 
 #include <assert.h>
 
-#include <jack/types.h"
-#include <jack/engine.h>
+#include <jack/types.h>
+#include "engine.h"
 #include <sysdeps/time.h>
 
 #include "include/internal.h"
--- jack1-0.124.1/jackd/jackstart.c.orig	2014-01-22 21:25:41.000000000 +0100
+++ jack1-0.124.1/jackd/jackstart.c	2014-02-01 15:29:39.569054121 +0100
@@ -37,7 +37,7 @@
 #undef _POSIX_SOURCE
 #include <sys/capability.h>
 
-#include "jack/start.h"
+#include "start.h"
 #include "md5.h"
 #include "jack_md5.h"
 
--- jack1-0.126.0/libjack/client.c.orig	2023-03-28 18:25:33.333465624 +0200
+++ jack1-0.126.0/libjack/client.c	2023-03-28 18:26:40.826433316 +0200
@@ -2439,7 +2439,7 @@ jack_activate (jack_client_t *client)
 		 */
 
 		req.type = SetClientCapabilities;
-		req.x.client_id = client->control->id;
+		jack_uuid_copy (&req.x.client_id, client->control->uuid);
 		req.x.cap_pid = client->control->pid;
 
 		jack_client_deliver_request (client, &req);
