--- jack2-1.9.12/wscript.orig	2017-12-13 22:19:04.000000000 +0100
+++ jack2-1.9.12/wscript	2017-12-22 17:20:35.424934055 +0100
@@ -416,7 +416,7 @@
     alsa.add_package('alsa', atleast_version='1.0.18')
     firewire = add_auto_option(opt, 'firewire', help='Enable FireWire driver (FFADO)', conf_dest='BUILD_DRIVER_FFADO')
     firewire.add_package('libffado', atleast_version='1.999.17')
-    freebob = add_auto_option(opt, 'freebob', help='Enable FreeBob driver')
+    freebob = add_auto_option(opt, 'freebob', help='Enable FreeBob driver', conf_dest='BUILD_DRIVER_FREEBOB')
     freebob.add_package('libfreebob', atleast_version='1.0.0')
     iio = add_auto_option(opt, 'iio', help='Enable IIO driver', conf_dest='BUILD_DRIVER_IIO')
     iio.add_package('gtkIOStream', atleast_version='1.4.0')
