--- jack-audio-connection-kit-0.98.1/configure.in.orig	2004-04-25 15:41:25.000000000 +0200
+++ jack-audio-connection-kit-0.98.1/configure.in	2004-08-25 18:27:29.936086688 +0200
@@ -449,7 +449,7 @@
 # the cached result to work around this problem.
 READLINE_DEPS=""
 HAVE_READLINE=true
-AC_CHECK_LIB(readline, readline, [],
+AC_CHECK_LIB(readline, readline, [:],
     [unset ac_cv_lib_readline_readline
      AC_CHECK_LIB(readline, readline, [READLINE_DEPS="-ltermcap"],
         [unset ac_cv_lib_readline_readline
