--- jack-audio-connection-kit-0.102.20/configure.ac.orig	2006-08-31 16:17:10.000000000 +0200
+++ jack-audio-connection-kit-0.102.20/configure.ac	2006-10-09 23:24:52.507574000 +0200
@@ -208,9 +208,7 @@
 AM_CONDITIONAL(USE_POSIX_SHM, $USE_POSIX_SHM)
 
 JACK_CORE_CFLAGS="-I\$(top_srcdir)/config -I\$(top_srcdir) \
--I\$(top_srcdir) -D_REENTRANT -D_POSIX_PTHREAD_SEMANTICS -Wall -g"
-JACK_OPT_CFLAGS="$JACK_CORE_CFLAGS -march=pentium2 -mcpu=pentium4 -O3 \
--ffast-math -funroll-loops -fprefetch-loop-arrays"
+-I\$(top_srcdir) -D_REENTRANT -D_POSIX_PTHREAD_SEMANTICS -Wall"
 JACK_CFLAGS="$JACK_CORE_CFLAGS $CFLAGS"
 
 dnl
@@ -338,12 +336,16 @@
     fi
 
     AC_DEFINE(x86, 1, "Nope it's intel")
-    COMMON_X86_OPT_FLAGS="-DREENTRANT -O3 -fomit-frame-pointer -ffast-math -funroll-loops"
+    COMMON_X86_OPT_FLAGS="-DREENTRANT -O3 -fomit-frame-pointer -ffast-math -funroll-all-loops -fforce-addr -fprefetch-loop-arrays"
     
-    if test "$target_cpu" = "i586"; then
+    if test "$target_cpu" = "i486"; then
+	JACK_OPT_CFLAGS="-march=i486 "
+    elif test "$target_cpu" = "i586"; then
 	JACK_OPT_CFLAGS="-march=i586 "
     elif test "$target_cpu" = "i686"; then
 	JACK_OPT_CFLAGS="-march=i686"
+    elif test "$target_cpu" = "athlon"; then
+        JACK_OPT_CFLAGS="-march=athlon"
     elif test "$target_cpu" = "x86_64"; then
 	JACK_OPT_CFLAGS="-march=k8"
     else
