--- jack-audio-connection-kit-0.100.0/drivers/alsa/alsa_driver.c.orig	2005-11-25 12:30:41.000000000 +0000
+++ jack-audio-connection-kit-0.100.0/drivers/alsa/alsa_driver.c	2005-11-25 12:31:12.743735832 +0000
@@ -1667,7 +1667,7 @@
 	return 0;
 }
 
-static inline int
+static int
 alsa_driver_run_cycle (alsa_driver_t *driver)
 {
 	jack_engine_t *engine = driver->engine;
--- jack1-0.126.0/drivers/dummy/dummy_driver.c.orig	2023-03-28 18:22:54.887657332 +0200
+++ jack1-0.126.0/drivers/dummy/dummy_driver.c	2023-03-28 18:24:19.230533741 +0200
@@ -213,7 +213,7 @@ static int dummy_driver_nt_start (dummy_
 }
 #endif
 
-static inline int
+static int
 dummy_driver_run_cycle (dummy_driver_t *driver)
 {
 	jack_engine_t *engine = driver->engine;
