--- jack2-1.9.20/man/wscript.orig	2022-01-15 16:09:56.000000000 +0100
+++ jack2-1.9.20/man/wscript	2022-03-18 20:03:45.439254379 +0100
@@ -5,6 +5,6 @@
 def build(bld):
     bld.exec_command('cd man ; sh fill_template {} {}'.format(
         bld.env['JACK_VERSION'],
-        bld.env['HAVE_JACK_EXAMPLE_TOOLS'])
+        bld.env['BUILD_JACK_EXAMPLE_TOOLS'])
     )
     bld.install_files(bld.env['MANDIR'], bld.path.ant_glob('*.1'))
