--- jack-audio-connection-kit/configure.ac.orig	2005-04-15 23:27:15.000000000 +0200
+++ jack-audio-connection-kit/configure.ac	2005-05-03 19:57:21.000000000 +0200
@@ -160,9 +160,7 @@
 AM_CONDITIONAL(USE_POSIX_SHM, $USE_POSIX_SHM)
 
 JACK_CORE_CFLAGS="-I\$(top_builddir)/config -I\$(top_srcdir) \
--I\$(top_builddir) -D_REENTRANT -D_POSIX_PTHREAD_SEMANTICS -Wall -g"
-JACK_OPT_CFLAGS="$JACK_CORE_CFLAGS -march=pentium2 -mcpu=pentium4 -O3 \
--ffast-math -funroll-loops -fprefetch-loop-arrays"
+-I\$(top_builddir) -D_REENTRANT -D_POSIX_PTHREAD_SEMANTICS -Wall"
 JACK_CFLAGS="$JACK_CORE_CFLAGS $CFLAGS"
 
 dnl
@@ -254,12 +252,16 @@
   fi
 
   AC_DEFINE(x86, 1, "Nope it's intel")
-  COMMON_X86_OPT_FLAGS="-DREENTRANT -O3 -fomit-frame-pointer -ffast-math -funroll-loops -fmove-all-movables"
+  COMMON_X86_OPT_FLAGS="-DREENTRANT -O3 -fomit-frame-pointer -ffast-math -funroll-all-loops -fforce-addr -fprefetch-loop-arrays"
 
-  if test "$target_cpu" = "i586"; then
-      JACK_OPT_CFLAGS="-march=i586 "
+  if test "$target_cpu" = "i486"; then
+      JACK_OPT_CFLAGS="-march=i486"
+  elif test "$target_cpu" = "i586"; then
+      JACK_OPT_CFLAGS="-march=i586"
   elif test "$target_cpu" = "i686"; then
       JACK_OPT_CFLAGS="-march=i686"
+  elif test "$target_cpu" = "athlon"; then
+      JACK_OPT_CFLAGS="-march=athlon"
   else
       :
   fi
@@ -267,7 +269,9 @@
   JACK_OPT_CFLAGS="$COMMON_X86_OPT_FLAGS $JACK_OPT_CFLAGS $MMX_FLAGS $SSE_FLAGS"
 
 elif test "$target_cpu" = "x86_64"; then
-      JACK_OPT_CFLAGS="-DREENTRANT -O3 -march=k8 -fomit-frame-pointer -ffast-math -funroll-loops -fmove-all-movables"
+      JACK_OPT_CFLAGS="$COMMON_X86_OPT_FLAGS $JACK_OPT_CFLAGS $MMX_FLAGS $SSE_FLAGS"
+elif test "$target_cpu" = "amd64"; then
+      JACK_OPT_CFLAGS="$COMMON_X86_OPT_FLAGS $JACK_OPT_CFLAGS $MMX_FLAGS $SSE_FLAGS"
 fi
 
 AC_ARG_ENABLE(optimize,
