- fix for "Unresolved symbols found"
--- jack-audio-connection-kit-0.121.3/jackd/Makefile.am.orig	2011-09-28 14:27:47.000000000 +0200
+++ jack-audio-connection-kit-0.121.3/jackd/Makefile.am	2012-11-11 17:13:29.103111610 +0100
@@ -51,7 +51,7 @@
         ../libjack/midiport.c ../libjack/ringbuffer.c ../libjack/shm.c \
         ../libjack/thread.c ../libjack/time.c  ../libjack/transclient.c \
         ../libjack/unlock.c
-libjackserver_la_LIBADD  = simd.lo @OS_LDFLAGS@ 
+libjackserver_la_LIBADD  = simd.lo @OS_LDFLAGS@ $(CAP_LIBS)
 libjackserver_la_LDFLAGS  = -export-dynamic -version-info @JACK_SO_VERSION@
 
 simd.lo: $(srcdir)/../libjack/simd.c
