diff -dur jack-1.9.10.orig/linux/alsa/JackAlsaDriver.cpp jack-1.9.10/linux/alsa/JackAlsaDriver.cpp
--- jack-1.9.10.orig/linux/alsa/JackAlsaDriver.cpp	2014-07-18 14:28:12.000000000 +0200
+++ jack-1.9.10/linux/alsa/JackAlsaDriver.cpp	2015-12-06 18:30:36.000000000 +0100
@@ -346,7 +346,7 @@
         fPlaybackChannels = ((alsa_driver_t *)fDriver)->playback_nchannels;
         return 0;
     } else {
-        JackAudioDriver::Close();
+        Close();
         return -1;
     }
 }
@@ -356,7 +356,9 @@
     // Generic audio driver close
     int res = JackAudioDriver::Close();
 
-    alsa_driver_delete((alsa_driver_t*)fDriver);
+    if (fDriver) {
+        alsa_driver_delete((alsa_driver_t*)fDriver);
+    }
 
     if (JackServerGlobals::on_device_release != NULL)
     {
